prow/github/app_auth_roundtripper.go

```diff
@@ -92,7 +92,7 @@ func (arr *appsRoundTripper) canonicalizedPath(url *url.URL) string {
 	return strings.TrimPrefix(url.Path, arr.hostPrefixMapping[url.Host])
 }
 
-var installationPath = regexp.MustCompile(`^/repos/(\w*)/(\w*)/installation$`)
+var installationPath = regexp.MustCompile(`^/repos/(\w|_|-|\.)+/(\w|_|-|\.)+/installation$`)
```

Comment:
How about just matching everything until the next slash, rather than "word or in this known character set"?

```suggestion
var installationPath = regexp.MustCompile(`^/repos/[^/]+/[^/]+/installation$`)
```

Reply:
That's a good idea!